token/confidential-transfer/proof-extraction/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "spl-token-confidential-transfer-proof-extraction"
+version = "0.1.0"
+description = "Solana Program Library Confidential Transfer Proofs"
+authors = ["Solana Labs Maintainers <maintainers@solanalabs.com>"]
+repository = "https://github.com/solana-labs/solana-program-library"
+license = "Apache-2.0"
+edition = "2021"
+
+[dependencies]
+solana-zk-sdk = "2.0.0"
+thiserror = "1.0.61"
+
+[dev-dependencies]
+spl-token-confidential-transfer-proof-generation = { version = "0.1.0", path = "../proof-generation" }
```

Comment:
This is up to you can be done later, but you could put all the tests in a separate crate to avoid having the extraction crate (the on-chain one) depend on the off-chain crate here.

Reply:
Good idea! I refactored the tests into a separate crate.